app/src/renderer/scripts/boot.js

```diff
@@ -0,0 +1,126 @@
+"use strict"
+/**
+ * Main module
+ * @module main
+ */
+
+import _Vue from "vue"
+import Router from "vue-router"
+import Tooltip from "vue-directive-tooltip"
+import Vuelidate from "vuelidate"
+import * as _Sentry from "@sentry/browser"
+import axios from "axios"
+
+import App from "../App"
+import routes from "../routes"
+import _Node from "../connectors/node"
+import _Store from "../vuex/store"
+import * as urlHelpers from "../../helpers/url.js"
+import _config from "../../config"
+
+/**
+ * Start the Vue app
+ */
+export const startApp = async (
+  config,
+  Node = _Node,
+  Store = _Store,
+  env = process.env,
+  Sentry = _Sentry,
+  Vue = _Vue
+) => {
+  /* istanbul ignore next */
+  Vue.config.errorHandler = (error, vm, info) => {
+    console.error(`An error has occurred: ${error}
+  
+  Guru Meditation #${info}`)
+
+    _Sentry.captureException(error)
+
+    if (store.state.devMode) {
+      throw error
+    }
+  }
+
+  /* istanbul ignore next */
+  Vue.config.warnHandler = (msg, vm, trace) => {
+    console.warn(`A warning has occurred: ${msg}
+  
+  Guru Meditation #${trace}`)
+
+    if (store.state.devMode) {
+      throw new Error(msg)
+    }
+  }
+
+  Vue.use(Router)
+  Vue.use(Tooltip, { delay: 1 })
+  Vue.use(Vuelidate)
+
+  // directive to focus form fields
+  /* istanbul ignore next */
+  Vue.directive(`focus`, {
+    inserted: function(el) {
+      el.focus()
+    }
+  })
+
+  // add error handlers in production
+  if (env.NODE_ENV === `production`) {
+    // Sentry is used for automatic error reporting. It is turned off by default.
+    Sentry.init({})
+
+    // this will pass the state to Sentry when errors are sent.
+    // this would also sent passwords...
+    // Sentry.configureScope(scope => {
```

Comment:
commented out?

Reply:
left in as a warning to not do this. see the comment